components/renderer/CaseStudy.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import LinkWrapper from '@components/LinkWrapper';
+import Image from 'next/image';
+import classNames from 'classnames';
+import { TypePageCaseStudy, TypePageFields } from '@types';
+import * as Contentful from 'contentful';
+
+type CaseStudyPropTypes = {
+    pageCaseStudy: Contentful.Entry<TypePageFields>;
+};
+
+const CaseStudy = ({ pageCaseStudy }: CaseStudyPropTypes) => {
+    const { title } = pageCaseStudy.fields;
+    const { fields: caseStudyPageFields } = pageCaseStudy.fields.content as TypePageCaseStudy;
+    const { summary, heroImage } = caseStudyPageFields.caseStudy.fields;
+    return (
+        <div
+            key={pageCaseStudy.fields.content.sys.id}
+            className="items-center block grid-cols-1 gap-0 mx-6 md:mx-0 md:grid md:grid-cols-14 pb-24"
+        >
+            <div className="col-span-4 col-start-2">
+                <h5
+                    className={classNames(`h5 overline`, {
```

Comment:
Remove add the classNames references and conditional classes. Just use `text-white`. It just makes it simpler.